planning/behavior_path_planner/src/scene_module/side_shift/manager.cpp

```diff
@@ -25,20 +25,22 @@
 namespace behavior_path_planner
 {
 
-SideShiftModuleManager::SideShiftModuleManager(
-  rclcpp::Node * node, const std::string & name, const ModuleConfigParameters & config)
-: SceneModuleManagerInterface(node, name, config, {})
+void SideShiftModuleManager::init(rclcpp::Node * node)
 {
+  // init manager interface
+  initInterface(node, {});
+
   SideShiftParameters p{};
 
+  std::string ns = "side_shift.";
```

Comment:
```suggestion
  const std::string ns = "side_shift.";
```

Reply:
and others

Reply:
@kosuke55 Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5771/commits/81332c9c3449ec0069c81597bd2435b5a439f83b.